apps/rewards/app/components/Panel/NewReward/NewReward.js

```diff
@@ -36,21 +37,25 @@ class NewReward extends React.Component {
 
   changeField = ({ target: { name, value } }) =>
     this.setState({ [name]: value })
-  
+
   onSubmit = () => {
-    const dataToSend = this.state
-    dataToSend.currency = currencies[this.state.amountCurrency]
+    const dataToSend = { ...this.state }
+    const BLOCK_DURATION = 15
```

Comment:
We should move the time handling to a helper function in a shared folder so we don't have to duplicate it.